.travis.yml

```diff
@@ -1,7 +1,8 @@
 sudo: false
 language: java
 jdk:
-  - oraclejdk7
+  - openjdk7
   - oraclejdk8
 script:
   - travis_wait mvn verify -Ponline-its -Dinvoker.streamLogs=true -DwlpVersion=8.5.5_07 -DwlpLicense=L-MCAO-9SYMVC
```

Comment:
Seems like a really old version.  I wonder if that is intentional?

Reply:
Actually, not sure that `-DwlpVersion=8.5.5_07 -DwlpLicense=L-MCAO-9SYMVC` are even used.  We are downloading the latest Liberty.